mmv1/templates/terraform/examples/sql_sqlserver_instance_backup.tf.erb

```diff
@@ -0,0 +1,16 @@
+# [START cloud_sql_sqlserver_instance_backup]
+resource "google_sql_database_instance" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]['sqlserver_instance_backup'] %>"
+  region           = "us-central1"
+  database_version = "SQLSERVER_2017_STANDARD"
+  root_password = "INSERT-PASSWORD-HERE"
+  settings {
+    tier = "db-custom-2-7680"
+    backup_configuration {
+      enabled                        = true
+      binary_log_enabled             = true
```

Comment:
Getting an API error in our tests: `Binary log can only be enabled for MySQL instances`